src/EntityAccessControlHandler.php

```diff
@@ -5,19 +5,36 @@
 use Drupal\Core\Access\AccessResult;
 use Drupal\Core\Entity\EntityAccessControlHandler as CoreEntityAccessControlHandler;
 use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityPublishedInterface;
+use Drupal\Core\Entity\EntityTypeInterface;
 use Drupal\Core\Session\AccountInterface;
 use Drupal\user\EntityOwnerInterface;
 
 /**
  * Controls access based on the generic entity permissions.
  *
  * @see \Drupal\entity\EntityPermissionProvider
+ *
+ * Note: Using this entity access controller makes your access cacheable per
```

Comment:
"Note: this access control handler will cause your pages to be cached per user." perhaps?

Reply:
:+1: